### PR TITLE
[DOC RouteInfo] Fix grammar, spelling, and formatting

### PR DESCRIPTION
**Pull request by tansongyang**

### CODE REVIEW

File: packages/@ember/-internals/routing/lib/system/transition.ts [hunk 1]
```diff
@@ -138,7 +138,7 @@
  * This property is a `RouteInfo` object that represents
  * where the router is transitioning to. It's important
  * to note that a `RouteInfo` is a linked list and this
- * property is simply represents leafmost route.
+ * property is simply the leafmost route.
```
Comment (locks) [inline packages/@ember/-internals/routing/lib/system/transition.ts]:
```suggestion
 * property represents the leafmost route.
```

File: packages/@ember/-internals/routing/lib/system/transition.ts [hunk 2]
```diff
@@ -147,8 +147,8 @@
  * This property is a `RouteInfo` object that represents
  * where transition originated from. It's important
  * to note that a `RouteInfo` is a linked list and this
- * property is simply represents head node of the list.
- * In the case of an initial render, from will be set to
+ * property is simply the head node of the list.
```
Comment (locks) [inline packages/@ember/-internals/routing/lib/system/transition.ts]:
```suggestion
 * property represents the head node of the list.
```

**Review by locks — CHANGES_REQUESTED** [review]
Some small adjustments. Thanks for your help!

**Comment by tansongyang** [comment]
@locks Thanks for your comments! I made the changes you suggested and also decided to just collapse everything into one commit.

**Review by locks — APPROVED** [review]
Looks great 👌

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**